neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -178,7 +190,9 @@ impl SendStream for WebTransportSendStream {
     fn send_data(&mut self, conn: &mut Connection, buf: &[u8]) -> Res<usize> {
         self.send(conn)?;
         if self.state == WebTransportSenderStreamState::SendingData {
+            self.bytes_written += buf.len() as u64;
```

Comment:
This will count rejected bytes, which isn't the intent of the statistic.

Reply:
I'll change this. Thanks.